src/client/app/components/kid-view/KidNote.jsx

```diff
@@ -19,45 +17,53 @@ class KidNote extends Component {
 	}
 
 	onSubmit() {
-		event.preventDefault();
-		var existingNotes;
-		if (this.state.notes == ''){
-			existingNotes = [];
+		if (this.state.note !== ''){
+			event.preventDefault();
+			var existingNotes = this.state.notes;
+			existingNotes.push(this.state.note);
+			this.setState({notes: existingNotes, note: '', placeholder: 'Type new note'});
 		} else {
-			existingNotes = this.state.notes;
+			this.setState({placeholder:'Note is required to submit'});
 		}
-		existingNotes.push(this.state.note);
-		this.setState({notes: existingNotes});
 	}
 
 	illustrateNotes() {
-		if (this.state.notes != ''){
-			return this.state.notes.map(note =>
-				<li>{note}</li>
-			);
-		}
+		return this.state.notes.map((note,index) =>
```

Comment:
Again you have to cover the case that there is nothing but an empty array to map over. i commented on this in more detail elsewhere